Resolves Issues 16 & 36----Past Orders

jeneve opened the pull request:
Adds feature for users to see an index of all orders they have placed

Adds feature for users to see a show page for a past order
   -user sees a link to each package on the order
   -user sees a total price for the order
   -user sees the status of the order
   -adds seed data so that the team can see all the new functionality in developement

all tests are passing, please review, comment or merge! @Jbern16 @marinacor1

jeneve committed:
merge master into self

jeneve committed:
remove notes

jeneve committed:
change test for past orders

jeneve committed:
WTF

jeneve committed:
leaving turing now

jeneve committed:
merged master into this branch

jeneve committed:
Adds user sees past order details functionality

~>change user can register test to pass after changing user factory

~>change user can log out test to pass after changing user factory

~>add user model tests- validations

~>add models test for Order, validates belongs to user

~>add test for past orders details

~>adds OrderPackage model and table

~>changed orders table to give needed methods

~>adds order show page feature

Jbern16 commented on space_age/app/models/order.rb:
```diff
@@ -0,0 +1,11 @@
+
+  def total_cost
+    packages.sum(:price)
+  end
```
in the future maybe this will be duplicate with trip total price

Jbern16 commented on space_age/app/views/orders/show.html.erb:
```diff
@@ -0,0 +1,12 @@
+  <% @order.packages.each do |package| %>
+    <li><%= package.title %> <%= link_to "show", package_path(package) %> Cost: <%= package.price %></li><br>
+  <% end %>
+</ul>
```
maybe we can use the cart template for this

Jbern16 commented on space_age/db/migrate/20160422114555_create_order_packages.rb:
```diff
@@ -0,0 +1,11 @@
+class CreateOrderPackages < ActiveRecord::Migration
+  def change
+    create_table :order_packages do |t|
+      t.references :user, index: true, foreign_key: true
```
maybe superfluous to store user id here? We will find out!

jeneve committed:
Adds seed data to create users, admins and orders
~>we three are all admins, usernames: jeneve, jon, and marina, passwords are "password"

jeneve committed:
fix to seeds

jeneve committed:
Adds more orders to each user to show full functionality

Jbern16 commented:
Looks good. The functionality looks production quality to me.
